docs/modules/reference/pages/logging/introduction.adoc

```diff
@@ -0,0 +1,90 @@
+[[ref-logging]]
+= Logging
+
+== List of Log Files
+
+[options="header"]
+[cols="1,3"]
+|===
+| Name
+| Description
+
+| ackd.log
+|
+
+| alarmd.log
+|
+
+| bsmd.log
+|
+
+| [[ref-logging-collectd.log]] collectd.log 
```

Comment:
Don't need an anchor defined here
```suggestion
| collectd.log 
```

Reply:
There shouldn't be anchors inside a table.  Plus it doesn't make sense to link to a row in the table.